src/smartAuthorization.ts

```diff
@@ -0,0 +1,3 @@
+export interface SmartAuthorization {
+    getSupportedSMARTScopes(): string[];
```

Comment:
make async and add a comments explaining method

Reply:
added